scripts/update-pot.sh

```diff
@@ -49,7 +49,7 @@ msgcat ./pot/*.pot -s -o errpages.pot.new &&
 
 ## Update all existing dictionaries with the new content ...
 for f in `ls -1 ./*.po` ; do
-	echo -n "Update: ${f} ... "
+	printf "Update: ${f} ... "
```

Comment:
Here and in several similar places where external input becomes a part of the printf format string, please use a "%s" or a similar format string instead. I am aware that the probability of a file name or similar input containing `%` or other special characters is small in these contexts. Still we should not abuse the format string like that.